packages/aws-client/src/S3.ts

```diff
@@ -242,7 +242,6 @@ export const waitForObjectToExist = async (params: {
 * @returns {Promise}
 **/
 export const s3PutObject = (params: PutObjectCommandInput) => s3().putObject({
-  ACL: 'private',
   ...params,
```

Comment:
pass in `params` as parameter

Reply:
I believe it is being passed in line 245 https://github.com/nasa/cumulus/pull/3310/files/f3c2eb49b72a61b4b18b4f5a12bccf6e25c81ef2#diff-d9978f7089c5ca4880d8c127632ce56e31605818748434b7e12a368bf7fc64f8R245

Reply:
> Do we still allow user to set the ACL? There are multiple places set ACL. Will they still work or there is alternative way to achieve the same access control? e.g. https://github.com/nasa/cumulus/blob/master/tf-modules/s3-replicator/index.js#L17

I tested with my own stack and disabled ACLs on my system bucket. I think this is not a breaking change because not specifying an ACL will still be accepted by buckets with bucket owner preferred setting https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html (see Object Ownership settings).